packages/components/tsconfig.json

```diff
@@ -21,6 +21,6 @@
 
 		"forceConsistentCasingInFileNames": true
 	},
-	"files": [ "src/index" ],
+	"include": [ "src" ],
```

Comment:
Why did this need to change? Storybook?

Reply:
Yep, specifically `react-docgen-typescript-loader` which is added by the [preset](https://github.com/storybookjs/presets/blob/master/packages/preset-typescript/index.js#L31).

Was there a reason we were including the single file? If its an issue I can create/extend the config specifically for Storybook.